packages/osd-optimizer/src/worker/webpack.config.ts

```diff
@@ -205,7 +205,9 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
         },
         {
           test: /\.(js|tsx?)$/,
-          exclude: /node_modules/,
+          // vega-lite and some dependencies don't have es5 builds
```

Comment:
Were `vega-label` and `vega-functions` sub-dependencies of `vega-lite` v5 that didn't have es5 builds? I see these are included in the lockfile but haven't been changed, so I'm wondering how these may have been failing to compile before.

Reply:
> Were vega-label and vega-functions sub-dependencies of vega-lite v5 that didn't have es5 builds? 

Yeah, I think that's more or less correct. It may also be that the import/require paths are a little different when importing from `src` rather than using the distributed build. But to be honest I'm not 100% sure on how this worked previously. @AMoo-Miki may have a more satisfying answer...

Reply:
We are importing a pre-built `vega-lite@4.17.0` and as a result, its dependencies were not being pulled in. With the newer v5.6.0, we are building from source and hence the deps are being pulled in.

Reply:
That makes sense - thanks! @joshuarrrr do you think it would be helpful to add that as a comment here, in case it will be a useful reference when doing the full upgrade from v4 -> v5?

Reply:
comment updated